### PR TITLE
Rename "proteins"->"amino acids" in protein-translation description table

### PR DESCRIPTION
**Pull request by 0xNeshi**
Closes https://github.com/exercism/problem-specifications/issues/1639

Related [forum discussion](https://forum.exercism.org/t/protein-translation-wording-should-be-changed-in-the-exercise-description/13157)

### CODE REVIEW

File: exercises/protein-translation/canonical-data.json [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
     {
       "uuid": "2c44f7bf-ba20-43f7-a3bf-f2219c0c3f98",
-      "description": "Empty RNA sequence results in no proteins",
+      "description": "Empty RNA sequence results in no amino acids",
       "comments": [
         "Tracks may choose to expect an error when an empty RNA sequence  ",
         "is given, or use this test to expect an empty result.            "
       ],
-      "property": "proteins",
+      "property": "protein",
```
Comment (0xNeshi) [inline exercises/protein-translation/canonical-data.json:11]: The function returns a _single_ protein, not multiple ones.
An alternative name here would be `polypeptide`, but that would maybe require changing the whole exercise to be focused on polypeptide translation, so I left it as-is.

File: exercises/protein-translation/description.md [hunk 2]
```diff
@@ -9,27 +9,29 @@ RNA: `"AUGUUUUCU"` => translates to
...
 All subsequent codons after are ignored, like this:
 
 RNA: `"AUGUUUUCUUAAAUG"` =>
 
 Codons: `"AUG", "UUU", "UCU", "UAA", "AUG"` =>
 
-Protein: `"Methionine", "Phenylalanine", "Serine"`
+Polypeptide: `"Methionine", "Phenylalanine", "Serine"`
 
 Note the stop codon `"UAA"` terminates the translation and the final methionine is not translated into the protein sequence.
```
Comment (0xNeshi) [inline exercises/protein-translation/description.md:30]:
```suggestion
Note the stop codon `"UAA"` terminates the translation and the final methionine is not translated into the polypeptide sequence.
```
Not sure how accurate we want to be with regards to the technicalities.

File: exercises/protein-translation/canonical-data.json (hunk 1 above)
Comment (0xNeshi) [reply exercises/protein-translation/canonical-data.json:11]: The [CI is failing due to test-related changes.](https://github.com/exercism/problem-specifications/actions/runs/10979487489/job/30483946760?pr=2478) Should changes to the canonical data be done as part of a separate PR or not done at all?

Comment (Cool-Katt) [reply exercises/protein-translation/canonical-data.json:11]: As the [readme](https://github.com/exercism/problem-specifications?tab=readme-ov-file#test-data-canonical-datajson) says, tests that are already written in `canonical-data`  are immutable, meaning you'd have to generate new tests with new UUIDs if you want to change the `property` field.

Comment (IsaacG) [reply exercises/protein-translation/canonical-data.json:11]: Significant changes to the canonical data should first be discussed on the forum.

**Comment by IsaacG** [comment]
Would you mind starting a discussion on the forum about this first?

**Comment by 0xNeshi** [comment]
> Would you mind starting a discussion on the forum about this first?

I reverted all the changes not related to #1639 . Will open a discussion at a later point

**Comment by IsaacG** [comment]
> I reverted all the changes not related to #1639 . Will open a discussion at a later point

Exercism policy is to resolve a forum discussion _prior_ to opening a PR.

**Comment by Cool-Katt** [comment]
@0xNeshi - Open a discussion on the forum [here](https://forum.exercism.org/) with the changes you want to make (discussed in the issue you reference) and _then_  open a PR after that (or in this case you can reference this PR but for the future, please keep to the policy).

**Review by IsaacG — APPROVED** [review]

File: exercises/protein-translation/description.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ Note the stop codon `"UAA"` terminates the translation and the final methionine
 
 Below are the codons and resulting Amino Acids needed for the exercise.
```
Comment (BNAndras) [inline exercises/protein-translation/description.md:30]:
```suggestion
Below are the codons and resulting amino acids needed for the exercise.
```

**Review by Cool-Katt — COMMENTED** [review]
all good

**Review by Cool-Katt — APPROVED** [review]

**Review by BethanyG — APPROVED** [review]

**Review by BNAndras — APPROVED** [review]

**Review by iHiD — APPROVED** [review]

**Comment by 0xNeshi** [comment]
Just curios, what else is necessary to prompt the PR to get merged?

**Comment by BethanyG** [comment]
Looks like it needs  a `maintainers-admin` group member to approve.  iHiD approval should have done it (_IIRC it is he and Erik in that group_), but I think permissions have been a bit funky since we've re-arranged teams.

**Comment by 0xNeshi** [comment]
@exercism/maintainers-admin pinging you as this PR has been approved and ready to merge for almost 2 months now

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**